utilities/transactions/write_unprepared_transaction_test.cc

```diff
@@ -179,6 +179,8 @@ TEST_P(WriteUnpreparedTransactionTest, ReadYourOwnWrite) {
   iter->Prev();
   verify_state(iter, "a", "v7");
 
+  wup_txn->unprep_seqs_.clear();
```

Comment:
If this line was missing in the last patch, it means that its existence is not intuitive. If it is not intuitive, then an inline comment would help the future reader of why its existence is essential.